packages/connectivity-tests/test/atlas.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+set -e
+# To avoid username and password variables showing up in the logs
+set +x
+
+USERNAME="${CONNECTIVITY_TEST_ATLAS_USERNAME}"
+PASSWORD="${CONNECTIVITY_TEST_ATLAS_PASSWORD}"
+HOSTNAME="${CONNECTIVITY_TEST_ATLAS_HOSTNAME}"
+DATA_LAKE_HOSTNAME="${CONNECTIVITY_TEST_DATA_LAKE_HOSTNAME}"
+
+if
+  [[ -z "${USERNAME}" ]] ||
+    [[ -z "${PASSWORD}" ]] ||
+    [[ -z "${HOSTNAME}" ]] ||
+    [[ -z "${DATA_LAKE_HOSTNAME}" ]]
+then
+  echo "Atlas credentials are not provided"
+  exit 0
```

Comment:
Do we want to fail here, at least if it’s CI (evergreen should set `IS_CI` in the environment)?

Reply:
Oh, yeah, good catch! Definitely should fail in CI and I think it would be better to just skip locally if that sounds good to you

Reply:
Yup, that sounds good :)